core/domain/user_services.py

```diff
@@ -69,18 +72,22 @@ class UserSettings(object):
         preferred_site_language_code: str or None. System language preference.
     """
     def __init__(
-            self, user_id, email, username=None, last_agreed_to_terms=None,
+            self, user_id, email, username=None,
+            role=feconf.ROLE_EXPLORATION_EDITOR, last_agreed_to_terms=None,
```

Comment:
As mentioned, please remove default arg from `role` argument, and force callers to specify it explicitly. Make sure to ensure that all calls to this method in the codebase are changed accordingly.